### PR TITLE
Scylla config template

### PR DESCRIPTION
**Pull request by ulenger**
Restart and startup handler added
scylla.yaml template for single node with cluster name var added

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+# handler file for system restart
+- name: start scylla-server
+  service: name=scylla-server.service state=started enabled=yes
```
Comment (jwstric2) [inline handlers/main.yml]: Can we convert these to Indented blocks format.  While acceptable, most of our ansible items have used idented blocks as well as its trending with most ansible doc examples showing this too.

```
name: start scylla-server
service:
  name: scylla-server.service
  start: started
  enabled: yes
```

File: handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- name: restart scylla-server
```
Comment (jwstric2) [inline handlers/main.yml:4]: Same as above, indented blocks if possible..

File: tasks/setup-RedHat.yml [hunk 3]
```diff
@@ -15,3 +20,4 @@
   package:
     name: "{{ scylladb_package_name }}"
     state: present
+  notify: "start scylla-server"
```
Comment (jwstric2) [inline tasks/setup-RedHat.yml]: why not just notifiy restart scylla... would seem a new package would restart it but if not we would want to restart it here.

File: handlers/main.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+# handler file for system restart
+- name: start scylla-server
```
Comment (jwstric2) [inline handlers/main.yml]: Can all the functionality be moved to just the "restart handle", can set state to enabled, ect.. in there?   Most opensource playbook I've seen have just restart handle, especially looking at long term infra where changes needs to really call a restart vs start.

File: templates/scylla.yaml.j2 [hunk 5]
```diff
@@ -0,0 +1,504 @@
...
+# not met, performance and reliability can be degraded.
+#
+# These requirements include:
+#    - A filesystem with good support for aysnchronous I/O (AIO). Currently,
+#      this means XFS.
+#
+# false: strict environment checks are in place; do not start if they are not met.
+# true: relaxed environment checks; performance and reliability may degraade.
+#
+developer_mode: {{ scylladb_developer_mode | default('false') }}
```
Comment (jwstric2) [inline templates/scylla.yaml.j2]: Our default is set to "true" in the main.yml.  I'm fine if we want to leave the default in the main.yml without adding a default here.

File: templates/scylla.yaml.j2 [hunk 6]
```diff
@@ -0,0 +1,504 @@
...
+# Scylla will only read and use the first segment
+#######################################
+
+### Supported Parameters
+
+# The name of the cluster. This is mainly used to prevent machines in
+# one logical cluster from joining another.
+# It is recommended to change the default value when creating a new cluster.
+# You can NOT modify this value for an existing cluster
+cluster_name: {{ scylladb_cluster_name | default('Test Cluster') }}
```
Comment (jwstric2) [inline templates/scylla.yaml.j2:19]: Add default to YAML?

File: defaults/main.yml [hunk 7]
```diff
@@ -1,3 +1,10 @@
 ---
 scylladb_repo_url: ""
 scylladb_version: ""
+scylladb_cluster_name: ""
+scylladb_seed_servers: ""
```
Comment (jwstric2) [inline defaults/main.yml]: Don't think we are using this?

File: defaults/main.yml [hunk 8]
```diff
@@ -1,3 +1,10 @@
 ---
 scylladb_repo_url: ""
 scylladb_version: ""
+scylladb_cluster_name: ""
```
Comment (jwstric2) [inline defaults/main.yml:4]: Add documentation to README.md for this.

File: defaults/main.yml [hunk 9]
```diff
@@ -1,3 +1,10 @@
...
+scylladb_seed_servers: ""
+scylladb_developer_mode: "true" # This should be set to false for production
```
Comment (jwstric2) [inline defaults/main.yml:5]: Documentation to README.md for this.

File: defaults/main.yml [hunk 10]
```diff
@@ -1,3 +1,10 @@
...
+
+scylladb_jmx:
```
Comment (jwstric2) [inline defaults/main.yml]: Do we have to touch this file / config?  These seem like defaults to me unless I'm incorrect.

File: templates/scylla-jmx.j2 [hunk 11]
```diff
@@ -0,0 +1,3 @@
+{% for key,value in scylladb_jmx|dictsort(by="key") %}
```
Comment (jwstric2) [inline templates/scylla-jmx.j2]: Not sure if we want this key,value items.  I'm not opposed to it, but it's unclear what goes in here.  It's also nice to have defaults, stuff people will not need to touch (or should).  Is this file needed at was a question at the top related to the key val item.

File: defaults/main.yml (hunk 8 above)
Comment (ulenger) [reply defaults/main.yml:4]: Done

File: defaults/main.yml (hunk 7 above)
Comment (ulenger) [reply defaults/main.yml]: We keep scylladb_cluster_name

File: templates/scylla.yaml.j2 (hunk 5 above)
Comment (ulenger) [reply templates/scylla.yaml.j2]: Agree

File: handlers/main.yml (hunk 4 above)
Comment (ulenger) [reply handlers/main.yml]: Done, using restart handler

File: tasks/setup-RedHat.yml (hunk 3 above)
Comment (ulenger) [reply tasks/setup-RedHat.yml]: Done

File: handlers/main.yml (hunk 2 above)
Comment (ulenger) [reply handlers/main.yml:4]: Done

File: handlers/main.yml (hunk 1 above)
Comment (ulenger) [reply handlers/main.yml]: Removed from file, we will only use restart handler

File: defaults/main.yml (hunk 9 above)
Comment (ulenger) [reply defaults/main.yml:5]: Removed seed server var and provided info on cluster_name and developer_mode

File: defaults/main.yml (hunk 10 above)
Comment (ulenger) [reply defaults/main.yml]: Removed

File: templates/scylla-jmx.j2 (hunk 11 above)
Comment (ulenger) [reply templates/scylla-jmx.j2]: Removed

**Review by jwstric2 — APPROVED** [review]
Looks good, thanks
